examples/FirebaseRoom_ESP8266/FirebaseRoom_ESP8266.ino

```diff
@@ -20,23 +20,24 @@
 #include <ESP8266WiFi.h>
 #include <FirebaseArduino.h>
 
+const int pin_grove = 15;
```

Comment:
can you use camelCase like the other arduino sample?
https://www.arduino.cc/en/Tutorial/Button